### ISSUE TITLE
in-use volume extend support

### ISSUE THREAD

**Issue by ShiroDN** [LINBIT/openstack-cinder#10]
Hi, it seems that the in-use volume extension doesn't work even though it should work according to docs https://linbit.com/drbd-user-guide/linstor-guide-1_0-en/#_using_drbd_as_the_transport_protocol  ("Resizing of attached and in-use volumes does not fully work. While the resize itself is successful, the compute service will not propagate it to the VM until after a restart.").

After some digging, it seems it is because you use LocalConnector https://github.com/openstack/os-brick/blob/master/os_brick/initiator/connectors/local.py that doesn't support volume extend.

Are you planning to implement a live extension - possibly in a new connector?

**Comment by WanzenBug** [comment]
> Are you planning to implement a live extension - possibly in a new connector?

Not currently, no. The effort needed to get this implemented is not justified by the level of adoption we see.

If someone were to implement the "TODO" on the LocalConnector, it should just start working, so I'd say "patches welcome", but thats for the os-brick maintainers.

**Comment by Nils98Ar** [comment]
@WanzenBug Is this still the case?

**Comment by WanzenBug** [comment]
Yes. The os_brick connecter still does not allow "local" volumes to be extended online

**Comment by Zeratyl06** [comment]
Has the [commit](https://github.com/LINBIT/openstack-cinder/commit/ef8e1f37c79c4bf2cf75f1884e2a22b717d791c2) solved this issue?

**Comment by WanzenBug** [comment]
It is solved if you also use the corresponding [os-brick](https://github.com/LINBIT/openstack-os-brick) and [nova](https://github.com/LINBIT/openstack-nova) branches. Still not much movement upstream :/
